src/utilities/sequence-editor/tree-utils.ts

```diff
@@ -46,9 +46,10 @@ export function getFromAndTo(nodes: (SyntaxNode | null)[]): { from: number; to:
   );
 }
 
-export function getAncestorNode(node: SyntaxNode | null, name: string) {
+export function getAncestorNode(node: SyntaxNode | null, ...names: string[]) {
```

Comment:
Can this function name/variables be renamed in such a way that is easier to understand at a glance? When it was a single `name` string, it was a little easier to figure out that it's traversing upward until it finds the target node name. Now that it's an array of names, I'm not too sure what the purpose is. The current way that this is being used is by 
```
commandNode = getAncestorNode(
    node,
    TOKEN_COMMAND,
    TOKEN_ACTIVATE,
    TOKEN_GROUND_BLOCK,
    TOKEN_GROUND_EVENT,
    TOKEN_LOAD,
  );
```
Does this mean that all these names qualify as a `commandNode`? Can we maybe also write a `getCommandNodeParent` function that has these names baked into the function?